services/QuillLMS/app/queries/progress_reports/standards/all_classrooms_standard.rb

```diff
@@ -11,7 +11,10 @@ def results(classroom_id, student_id)
       <<-SQL
         WITH final_activity_sessions AS (
           SELECT
-            activity_sessions.*,
+            activity_sessions.activity_id,
+            activity_sessions.id,
+            activity_sessions.timespent,
+            activity_sessions.user_id,
```

Comment:
Looks like you need to pull `activity_sessions.percentage` for the subquery in the main query.

Reply:
Thank you.  I'm noticing I need it on the other query too.

Reply:
FYI.  I manually tested the spec failures locally since circleci is currently down